easybuild/easyconfigs/x/xhmm/xhmm-1.0-foss-2016b.eb

```diff
@@ -0,0 +1,29 @@
+#  easybuild easyconfig
+#
+#  John Dey  jfdey@fredhutch.org
+#
+#  Fred Hutchinson Cancer Research Center
+
+easyblock = 'MakeCp'
+
+name = 'xhmm'
+version = '1.0'
+
+homepage = 'https://bitbucket.org/statgen/xhmm/'
+description = """The XHMM (eXome-Hidden Markov Model) C++ software suite calls copy number variation (CNV) from next-generation sequencing projects, where exome capture was used (or targeted sequencing, more generally). Specifically, XHMM uses principal component analysis (PCA) normalization and a hidden Markov model (HMM) to detect and genotype copy number variation (CNV) from normalized read-depth data from targeted sequencing experiments.""" 
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://bitbucket.org/statgen/xhmm/get']
+sources = ['cc14e528d909.zip']
+
+start_dir = 'statgen-xhmm-cc14e528d909'
```

Comment:
@fizwit we should avoid hardcoding the commit ID in multiple places... maybe a `prebuildopts = "cd statgen-xhmm*/ &&"` is better in this case?

Also, is this really needed? This should only be required if unpacking the tarball results in more than just this directory...